salt/top.sls

```diff
@@ -10,10 +10,19 @@ base:
     - rebootmgr
     - transactional-update
     - haproxy
+    - kubectl-config
```

Comment:
Can this go in a group `roles:kube-(master|minion)`? Does the admin need this config?

Reply:
Actually, while it's unrelated, it's really nice to have the config there - kubectl works on the admin node all of a sudden ;)